packages/containers/src/AppLoader/AppLoader.md

```diff
@@ -0,0 +1,47 @@
+# App Loader
+
+This component is here to make http calls before the first render of the application. During the loading, it show the application's loader.
+
+## Where i put it ?
+
+On your top level component, you can place it this way :
+
+```html
+<AppLoader>
+    <div>
+        <ShortcutManager view="shortcuts" />
+        <IconsProvider icons={mockIconDefinition} />
+        <ACKDispatcher />
+        <Notification />
+        {props.children}
+    </div>
+</AppLoader>
+```
+
+Another way to do it could be to use the inject placeholders to inject all the stuff
```

Comment:
Can you add some code here for example, so we don't have to go in the code to get the placeholder names or go over inject's code to know how to use it ?